src/base/DelegateInputLabelMixin.js

```diff
@@ -31,18 +32,42 @@ export default function DelegateInputLabelMixin(Base) {
       this[setState]({ ariaLabel });
     }
 
+    // Forward ARIA labelledby to the input element.
+    get ariaLabelledBy() {
+      return this[state].ariaLabelledBy;
+    }
+    set ariaLabelledBy(ariaLabelledBy) {
```

Comment:
@JanMiksovsky the thing I'm stuck on is for some reason when aria-labelledby is set as an attribute in the HTML - it doesn't trigger setting the ariaLabelledby property

ie <elix-input aria-labelledby="foo"></elix-input> doesn't call the set function

Reply:
AttributeMarshallingMixin converts an attribute to a property name by looking for hyphens, removing them, and converting the following letter to uppercase. Since the `aria-labelledby` has only one hyphen, I think it's expecting the property name to be `ariaLabelledby` with a lowercase "b".

Reply:
![image](https://user-images.githubusercontent.com/8482334/86203346-a4b31e00-bb19-11ea-85ad-362248bf0787.png)

I didn't even think of that haha, thanks!